frontend/wallet/src/render/views/settings/StakingSettings.re

```diff
@@ -73,7 +73,7 @@ module AccountInfo = [%graphql
 module AccountInfoQuery = ReasonApollo.CreateQuery(AccountInfo);
 
 [@react.component]
-let make = (~publicKey, ~stakingActive=false) => {
+let make = (~publicKey, ~stakingActive as _=false) => {
```

Comment:
This seems suspicious but we can probably leave it until the next pass on the settings